tests/scattering1d/test_utils_scattering1d.py

```diff
@@ -10,15 +10,15 @@ def test_compute_padding():
     Test the compute_padding function
     """
 
-    pad_left, pad_right = compute_padding(5, 16)
+    pad_left, pad_right = compute_padding(32, 16)
     assert pad_left == 8 and pad_right == 8
 
     with pytest.raises(ValueError) as ve:
-        _, _ = compute_padding(3, 16)
+        _, _ = compute_padding(8, 16)
     assert "should be larger" in ve.value.args[0]
 
     with pytest.raises(ValueError) as ve:
-        _, _ = compute_padding(6, 16)
+        _, _ = compute_padding(64, 16)
```

Comment:
Why increase?

Reply:
these are the same tests, but with the first argument being the exponent of 2.
before: J_pad=5, N_input=16
after: N=2**J_pad=32, N_input=16

etc.